src/Tools/dotnet-monitor/Profiler/ProfilerService.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using Microsoft.Diagnostics.NETCore.Client;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.Profiler
+{
+    internal sealed class ProfilerService : BackgroundService
+    {
+        private static readonly Guid ProfilerClsId = new Guid("6A494330-5848-4A23-9D87-0E57BBF6DE79");
```

Comment:
It looks like we use this specific GUID in ProfilerHelper.cs; even if we can't reuse the value and have to copy-paste it here, do we want a comment saying these should not be changed independently of one another?